chemprop/utils.py

```diff
@@ -61,31 +63,31 @@ def save_checkpoint(path: str,
         args = Namespace(**args.as_dict())
 
     state = {
-        'args': args,
-        'state_dict': model.state_dict(),
-        'data_scaler': {
-            'means': scaler.means,
-            'stds': scaler.stds
-        } if scaler is not None else None,
-        'features_scaler': {
-            'means': features_scaler.means,
-            'stds': features_scaler.stds
-        } if features_scaler is not None else None,
-        'atom_descriptor_scaler': {
-            'means': atom_descriptor_scaler.means,
-            'stds': atom_descriptor_scaler.stds
-        } if atom_descriptor_scaler is not None else None,
-        'bond_feature_scaler': {
-            'means': bond_feature_scaler.means,
-            'stds': bond_feature_scaler.stds
-        } if bond_feature_scaler is not None else None
+        "args": args,
+        "state_dict": model.state_dict(),
+        "data_scaler": {"means": scaler.means, "stds": scaler.stds} if scaler is not None else None,
+        "features_scaler": {"means": features_scaler.means, "stds": features_scaler.stds}
+        if features_scaler is not None
+        else None,
+        "atom_descriptor_scaler": {
+            "means": atom_descriptor_scaler.means,
+            "stds": atom_descriptor_scaler.stds,
+        }
+        if atom_descriptor_scaler is not None
+        else None,
+        "bond_feature_scaler": {
+            "means": bond_feature_scaler.means,
+            "stds": bond_feature_scaler.stds,
+        }
+        if bond_feature_scaler is not None
+        else None,
```

Comment:
The style of having the if/else be on the next line without indents is hard to read. Makes it look like those lines are separate arguments.

Reply:
Presumably this is the black formatting? Are there ways that we can override that when it causes problems if we set up pre-commits?

Reply:
this is just `black` doing it's best attempt to format the original block. Ternary statements should be simple one-liners, not complex dictionary constructions

Reply:
define them using `if/else` blocks before line 65 would be my suggestion